generators/hub/templates/package.json

```diff
@@ -1,27 +1,29 @@
 {
   "scripts": {
-    "build": "npm run lint && npm run clean && webpack --progress --colors --config webpack.config.js --output-path ./dist -p",
+    "build": "npm run lint && npm run clean && npm run webpack",
+    "webpack" : "webpack --progress --colors --config webpack.config.js --output-path ./dist -p",
     "postbuild": "npm run package",
     "package": "tfx extension create  --rev-version",
     "gallery-publish": "tfx extension publish --rev-version",
     "clean": "rimraf ./dist && rimraf ./*.vsix",
-    "lint": "tslint -c tslint.json \"src/*.ts"
+    "lint": "tslint -c tslint.json 'src/*.ts'"
```

Comment:
You can (and probably should) reference the tsconfig.json file here versus the file path to your TS source files. Think it'd tslint -c tslint.json -p tsconfig.json